### DISCUSSION TITLE
Why don't @tailwindcss/forms text fields and selects inherit the font-size and line-height of the parent

### DISCUSSION THREAD

**Discussion by sschultze** [tailwindlabs/tailwindcss#13556] [Help]
I notice that, when using `@tailwindcss/forms`, all text fields and selects are **not** using the `font-size` and `line-height` of their parent. Instead, they use the `baseFontSize` and `baseLineHeight`. This seems to be the case from the first published version on GitHub.

This seems quite counterintuitive for me - I would expect that such elements inherit the `font-size` and `line-height` of the parent: If a form on my website uses a specific text size, I want the contained text fields and selects to use this exact text size as well. So in my application, I override the corresponding default styles in `@tailwindcss/forms` with `font-size: inherit` and `line-height: inherit`.

**My question:** Is there any important technical reason I'm probably missing that have led the authors to apply `baseFontSize` and `baseLineHeight` to such elements? Or was it just a matter of taste?

**Comment by zulqasar** [comment]
Same concerns here. Have you found a solution?

**Reply by sschultze** [reply]
The workaround that I use is mentioned in my post above.
